core/record/recordExprValueConverter.js

```diff
@@ -0,0 +1,62 @@
+import * as R from 'ramda'
+
+import * as Survey from '@core/survey/survey'
+import * as NodeDef from '@core/survey/nodeDef'
+import * as Record from '@core/record/record'
+import * as Node from '@core/record/node'
+
+const _valueExprToValueBoolean = (survey, record, nodeCtx, valueExpr) =>
+  R.is(Boolean, valueExpr)
+    ? valueExpr
+    : R.is(String, valueExpr) && R.includes(valueExpr, ['true', 'false'])
+    ? valueExpr
+    : null
+
+const _isStringOrNumber = val => R.is(String, val) || R.is(Number, val)
```

Comment:
why having this function where it's used only by ```toPrimitive```?

Reply:
I'll remove it